appshell/appshell_extensions.cpp

```diff
@@ -276,32 +276,36 @@ class ProcessMessageDelegate : public ClientHandler::ProcessMessageDelegate {
                 ExtensionString filename = argList->GetString(1);
                 ExtensionString encoding = argList->GetString(2);
                 std::string contents = "";
+                bool preserveBOM = false;
```

Comment:
readfile is called from brackets and until then we don't know the encoding/bom of the file so default value is false.
so from brackets we don't need to send the argument